platform/v08/exit.go

```diff
@@ -0,0 +1,35 @@
+package v08
+
+import (
+	"github.com/buildpacks/lifecycle/platform/common"
+)
+
+var exitCodes = map[common.LifecycleExitError]int{
```

Comment:
This makes me regret the `platform` interface and trying to shovel all platform-related logic into there. But that's another story...